src/Router/BaseRouter.php

```diff
@@ -82,7 +82,9 @@ public function handle(ServerRequestInterface $request) : ResponseInterface
             $components = preg_split("/\/+?/", $path);
             $modulename = $components[0];
         }
-        if (is_dir($this->moduledir . "/" . $modulename)) {
+        if (is_dir($this->moduledir . "/" . $modulename)
+            || is_dir($this->projectdir . "/modules/" . $modulename)
+        ) {
```

Comment:
I'd prefer to get rid of the moduledir/projectdir hack and instead having a configurable module search path, but for 20.0 this is probably fine.